src/main/java/com/faforever/client/api/FafApiAccessorImpl.java

```diff
@@ -257,6 +257,7 @@ public void uploadMod(Path file, ByteCountListener listener) {
   @Override
   public void uploadMap(Path file, boolean isRanked, ByteCountListener listener) {
     MultiValueMap<String, Object> multipartContent = createFileMultipart(file, listener);
+    multipartContent.add("metadata", "{\"isRanked\":\"" + isRanked + "\"}");
```

Comment:
The 400 bad request issue #563 . With this line added I'm able to upload maps perfectly fine.